arcus_quickstart.md

```diff
@@ -0,0 +1,114 @@
+<!--
+author:   Arcus Education
+email:    arcus-support@chop.edu
+version:  1.0.0
+language: en
+narrator: US English Female
+title: Arcus Quickstart
+
+link:  https://cdn.jsdelivr.net/gh/arcus/Arcus_Labs_Orientation@main/assets/styles.css
+link:  https://cdn.jsdelivr.net/gh/arcus/education_modules@main/assets/styles.css
+script: https://kit.fontawesome.com/83b2343bd4.js
+script: https://cdn.jsdelivr.net/npm/mermaid@9.4.3/dist/mermaid.min.js
+-->
+
+# Arcus Labs: Quick Start
+
+Ready to start a research project with Arcus, or have you been given a computational lab and want to know what to do next? This quick start guide will jump start your Arcus journey! 
+
+If you have questions about Arcus Labs that aren't answered here, please see our [Arcus Lab User Guide]() for more detailed documentation. 
+
+## Start a research project with Arcus
+
+To work with data provided by Arcus, you'll need an Arcus Computational Lab (called an Arcus Lab for short).
+
+When you [request an Arcus Lab](https://support.arcus.chop.edu/servicedesk/customer/portal/6/create/307), we evaluate your proposed study to make sure it's a good fit for our service, then build the computational environment and provision the data for your team so you can start work. 
+This process takes time and coordination across multiple groups within Arcus. 
+Here's what you can do to make the process as smooth as possible: 
+
+- Check first: Is your proposed study a good fit for Arcus?
+- Save time: Hone your data request beforehand
+- Know the rules: Security, privacy, and the [Arcus Terms of Use](https://arcus.chop.edu/terms-of-use).
+
+### Is your proposed study a good fit for Arcus?
```

Comment:
Feedback from managers' meeting: swap it, so it doesn't feel like we're pointing fingers at them being not good enough, e.g. "Is Arcus a good fit for your study".